distribution/docker/src/docker/Dockerfile

```diff
@@ -29,7 +29,7 @@ ${source_elasticsearch}
 
 RUN tar zxf /opt/${elasticsearch} --strip-components=1
 RUN grep ES_DISTRIBUTION_TYPE=tar /usr/share/elasticsearch/bin/elasticsearch-env \
-    && sed -ie 's/ES_DISTRIBUTION_TYPE=tar/ES_DISTRIBUTION_TYPE=docker/' /usr/share/elasticsearch/bin/elasticsearch-env
+    && sed -i -e 's/ES_DISTRIBUTION_TYPE=tar/ES_DISTRIBUTION_TYPE=docker/' /usr/share/elasticsearch/bin/elasticsearch-env
```

Comment:
This change is to prevent `sed` creating a backup, which I think was unintentional. I noticed the file `elasticsearch-enve` being created in the Docker image.

Reply:
Great catch. Typically this confusion comes out of the sed version brought by macOS.